test/services/katello/applicability/applicable_content_helper_test.rb

```diff
@@ -21,20 +22,16 @@ def bound_repos(host)
 
         def setup
```

Comment:
The setup method does a lot of create vs save and I wonder if that's intentional?

Reply:
Not intentional, perhaps it could be related.